deploy-manage/security/secure-settings.md

```diff
@@ -16,232 +15,177 @@ mapped_pages:
+| {{ech}}              | Cloud UI                                  | Not available                          |
+| {{ece}}              | Cloud UI or API                                  | Not available                          |
+| {{eck}}              | Kubernetes `secrets`    | Kubernetes `secrets`                   |
+| Self-managed         | Manual configuration with [`elasticsearch-keystore`](elasticsearch://reference/elasticsearch/command-line-tools/elasticsearch-keystore.md) | Manual configuration with `kibana-keystore` |
+
+This section describes how to configure and manage secure settings in each keystore depending on the deployment model:
+* [{{es}} secure settings](./secure-settings.md#elasticsearch)
+* [{{kib}} secure settings](./secure-settings.md#kibana)
+
+## {{es}} secure settings [elasticsearch]
 
-## The {{es}} keystore
+The {{es}} keystore has some important characteristics and limitations to be aware of:
 
-With the {{es}} keystore, you can add a key and its secret value, then use the key in place of the secret value when you configure your sensitive settings.
+* **Only specific settings are allowed**: The keystore accepts only settings marked as *secure* in the [{{es}} configuration reference](elasticsearch://reference/elasticsearch/configuration-reference/index.md). Adding unsupported settings to the keystore causes the validation in the [_nodes/reload_secure_settings API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-nodes-reload-secure-settings) to fail and can also prevent {{es}} from starting.
+* **Mandatory for secure settings**: Settings marked as secure must be added to the keystore. They cannot be placed in `elasticsearch.yml` or set via environment variables. This differs from the {{kib}} keystore, which supports all settings.
+* **Changes require a restart**: Most secure settings take effect only after restarting the nodes. However, some are marked as *reloadable* and can be updated without a restart using the [reload_secure_settings](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-nodes-reload-secure-settings) API. Refer to [Reloadable settings](#reload-secure-settings) for more information.
+* **Keystore is per-node**: Each node in the cluster has its own keystore file. Secure settings must be specified individually on every node and must have the same values across the cluster. Orchestrated deployments, such as ECH, ECE, and ECK, handle this automatically when configuring secure settings.
+
+::::{tip}
+To verify whether a setting can be added to the keystore and whether it supports reloading without a restart, look for the `Secure` and `Reloadable` qualifiers in the [{{es}} security settings reference](elasticsearch://reference/elasticsearch/configuration-reference/security-settings.md). For example, you can search for `secure_bind_password` on that page.
+::::
```

Comment:
are the only secure settings security settings?

Reply:
looks like no: e.g. this monitoring setting `auth.secure_password`

https://docs-v3-preview.elastic.dev/elastic/elasticsearch/tree/main/reference/elasticsearch/configuration-reference/monitoring-settings

Reply:
Fixed ;)